vector/src/main/res/menu/menu_timeline.xml

```diff
@@ -19,32 +19,36 @@
         android:title="@string/action_invite"
         app:showAsAction="never" />
 
+    <!-- We always want to show this item as an icon -->
     <item
         android:id="@+id/video_call"
         android:icon="@drawable/ic_video"
         android:title="@string/action_video_call"
         android:visible="false"
         app:iconTint="?colorPrimary"
-        app:showAsAction="ifRoom"
+        app:showAsAction="always"
         tools:visible="true" />
 
+    <!-- We always want to show this item as an icon -->
     <item
         android:id="@+id/voice_call"
         android:icon="@drawable/ic_phone"
         android:title="@string/call"
```

Comment:
Maybe take the opportunity to use `action_voice_call` instead of `call` for parity. `action_voice_call` has the value "Voice call". This string can still be visible by the user when long pressing on the menu item.